.rustfmt.toml

```diff
@@ -1,6 +1,9 @@
 # These formatting rules to try conform the Substrate style guidelines:
 # >   https://wiki.parity.io/Substrate-Style-Guide
 
+comment_width = 80
+wrap_comments = true
```

Comment:
`comment_width = 80` <3
but to be honest I was having bad experience with `wrap_comments` in the past
with super ugly indented comments. but maybe I am just a bit too much looking
at the source code alignment.

Reply:
How do you align all your comments then? They usually look so neat and tidy. Something in your IDE? Could always increase that to 100 to mitigate the issue with indented comments.